presentational/Navbar.js

```diff
@@ -1,7 +1,102 @@
+import Link from 'next/link'
+import React from 'react'
+import { withRouter } from 'next/router'
 import styled from 'styled-components'
 
-const Navbar = styled.div`
-  border: 1px solid black;
+const NavWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  border: 2px;
```

Comment:
When I tried this branch there was no border. Maybe it should be `border: 2px solid` and some color.